TestAssemblies/AssemblyExplicitPropertyChanged/AssemblyExplicitPropertyChanged.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net48;net6.0</TargetFrameworks>
+    <TargetFrameworks>net48;net6</TargetFrameworks>
```

Comment:
Why are you switching to an unusual TFM? I've never seen `net6` being used anywhere.

Reply:
Maybe to test a bad csproj setting?

Reply:
It's not necessarily *bad* since it builds fine. Also, he made that change in 20 projects 🙂